SEUNAGBEYE/More-Recipes
# 153848212/refactor code for production

#### What does this PR do?

Refactored code to use production version of React when on production and add a webpack.prod.js file 

#### Description of Task to be completed?

Add coveralls code coverage service to generate a test report when the test is run and send it to coveralls

#### How should this be manually tested?
- download the react dev tool extension for your browser
- navigate to https://recipes-v1.herokuapp.com/ and click on the blue react icon to see the build type
#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#153848212

 - Refactor code for production

#### Screenshots
N/A

#### Questions:
N/A
